LuaRules/Configs/unit_targetting_metal_limit_defs.lua

```diff
@@ -0,0 +1,23 @@
+-- Value is the default state of the command
+local metalThreshholdsByUnitDefIDs = {
+	[UnitDefNames["hoverarty"].id] = {0, 0, 100, 300, 1000},
+	[UnitDefNames["cloaksnipe"].id] = {0, 0, 100, 300, 1000},
```

Comment:
Is this supposed to be the same table for every unit? In that case could do
```suggestion
local STATES = {0, 0, 100, 300, 1000} -- first value is the default state of the command
local metalThreshholdsByUnitDefIDs = {
	[UnitDefNames["hoverarty"].id] = STATES,
	[UnitDefNames["cloaksnipe"].id] = STATES,
```
etc

Reply:
That's due to me waffling back and forth over whether or not this should vary between units.

Eventually decided on simplicity.

Fixed now